release-info/howto.md

```diff
@@ -152,7 +152,7 @@ git push --tags
 
 ## 5-bis: Alternative for branching
 
-This needs to be more tested, tread with care.
+This needs to be more tested, treat with care.
```

Comment:
Ahem, please see thread at https://github.com/ocaml/ocaml/pull/9162#discussion_r353779455

Reply:
Damn,damn...

Reply:
When editing documentation and source comments, "maybe your best course is to tread lightly".  (Context: https://mashable.com/article/breaking-bad-bryan-cranston-quote)